src/screens/Dashboard/TripSearch.tsx

```diff
@@ -213,7 +218,20 @@ const TripSearch: React.FC<RootProps> = ({navigation}) => {
         rightButton={{type: 'chat'}}
         leftButton={{
           type: 'back',
-          onPress: () => navigation.goBack(),
+          onPress: () => {
+            if (callerRoute?.name) {
+              navigation.setParams({
+                callerRoute: undefined,
+              });
```

Comment:
Yes, but once we navigate back we want to reset what was set to not navigate back always to the map on the travel search.

Reply:
Hm, ok. Seems kind of hacky, but if it's needed, I guess that's fine 👍

Reply:
Does it fix this if we write the param as this `callerRoute: { name: string } | undefined` instead of `callerRoute?: { name: string }`? Because then those who doesn't provide `callerRoute` must send inn `undefined` for it, hopefully overwriting the existing params.